src/Shared/ParameterBindingMethodCache.cs

```diff
@@ -128,12 +131,18 @@ public bool HasTryParseMethod(ParameterInfo parameter)
             return _stringMethodCallCache.GetOrAdd(type, Finder);
         }
 
-        public Expression? FindBindAsyncMethod(ParameterInfo parameter)
+        public (Expression?, int) FindBindAsyncMethod(ParameterInfo parameter)
```

Comment:
```suggestion
        public (Expression? Expression, int ParamCount) FindBindAsyncMethod(ParameterInfo parameter)
```
Would this be better @davidfowl ?

Reply:
I don't think this changed much but it's fine